crates/dojo-erc/src/erc1155/components.cairo

```diff
@@ -0,0 +1,14 @@
+#[derive(Component, Copy, Drop, Serde)]
+struct OperatorApproval {
+    value: bool
+}
+
+#[derive(Component, Copy, Drop, Serde)]
+struct Balance {
```

Comment:
There's already the same component in the ERC20 module. We should extract that into a shared one and use that.

Reply:
I added a shared folder. I'm wondering if having shared and not shared components might not become a bit messy though :thinking:

Reply:
On second thought, I think you're right. Having each module in this crate encapsulated (i.e. not having shared components) feels cleaner.